<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeColorToneDialog</class>
 <widget class="QWidget" name="ChangeColorToneDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QSpinBox" name="spin_img_num">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_img">
       <property name="text">
        <string>Number:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Color Tone</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1" colspan="2">
           <widget class="QSlider" name="slider_red">
            <property name="minimum">
             <number>-255</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QSlider" name="slider_blue">
            <property name="minimum">
             <number>-255</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Green:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QSlider" name="slider_gray">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Time:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Frames</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QSlider" name="slider_green">
            <property name="minimum">
             <number>-255</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Blue:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Red:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spin_time">
            <property name="maximum">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Gray:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spin_red">
            <property name="minimum">
             <number>-255</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="spin_green">
            <property name="minimum">
             <number>-255</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSpinBox" name="spin_blue">
            <property name="minimum">
             <number>-255</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QSpinBox" name="spin_gray">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spin_img_num</tabstop>
  <tabstop>slider_red</tabstop>
  <tabstop>spin_red</tabstop>
  <tabstop>slider_green</tabstop>
  <tabstop>spin_green</tabstop>
  <tabstop>slider_blue</tabstop>
  <tabstop>spin_blue</tabstop>
  <tabstop>slider_gray</tabstop>
  <tabstop>spin_gray</tabstop>
  <tabstop>spin_time</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>button_ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>slider_red</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spin_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_green</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spin_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_blue</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spin_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_gray</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spin_gray</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_gray</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_gray</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
